src/main/java/ip/exception/DukeException.java

```diff
@@ -0,0 +1,11 @@
+package ip.exception;
+
+/**
+ * Exceptions thrown by Duke.
+ */
+public class DukeException extends Exception {
+
+    public DukeException() {
```

Comment:
No real need for this constructor right just like the IndexNotSpecified, unless I am wrong